packages/payload/src/index.ts

```diff
@@ -867,7 +867,7 @@ export const reload = async (
   if (payload.db.connect) {
     await payload.db.connect({ hotReload: true })
   }
-  global._payload_clientConfig = null
+  global._payload_localizedClientConfigs = {}
```

Comment:
The `localized` keyword here is a bit misleading as this is a reference to localization, not i18n.